enterprise/api/v1/views.py

```diff
@@ -173,6 +173,19 @@ def basic_list(self, request, *arg, **kwargs):
         serializer = self.get_serializer(queryset, many=True)
         return Response(serializer.data)
 
+    @action(detail=True)
```

Comment:
do we actually need this endpoint? is it ok to have this logic in the frontend? just curious since if the ui page is the only user of this endpoint I am thinking we should already have this info in the frontend using the `portalConfiguration` state?